packages/extension/package.json

```
@@ -73,6 +73,7 @@
   },
   "dependencies": {
     "@argent/x-sessions": "^4.8.5",
+    "@argent-x/ui": "*",
```

```suggestion
    "@argent/ui": "*",
```

Lets make it one argent